tests/content_hash/test_content_hash.py

```diff
@@ -0,0 +1,46 @@
+from src.caselawclient.content_hash import (content_hash, hashable_text,
+                                            validate_content_hash)
+
+VALID_DOC = b"""<?xml version="1.0" encoding="UTF-8"?>
+    <akomaNtoso
+      xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0"
+      xmlns:uk="https://caselaw.nationalarchives.gov.uk/akn">
+      <judgment name="judgment">
+        <meta>
+        <proprietary>
+        <uk:hash>c4367ebc0937f4dc2d6b372d9a09670e3606a5b3da77a070149755db5f942565</uk:hash>
+        <uk:cite>the meta section should not be present BAD</uk:cite>
+        </proprietary>
+        </meta>
+        <p>Do <b>use</b></p>
+        <p>this <i>valid</i> text</p>
+      </judgment>
+    </akomaNtoso>
+    """
+
+INVALID_DOC = VALID_DOC.replace(b"Do", b"Do not").replace(b"valid", b"invalid")
+
+
+def test_hashable_text():
+    """Do we correctly identify the text to hash, omitting the meta section and removing spaces?"""
+    assert hashable_text(VALID_DOC) == b"Dousethisvalidtext"
+    assert hashable_text(INVALID_DOC) == b"Donotusethisinvalidtext"
+    assert b"BAD" not in hashable_text(VALID_DOC)
```

Comment:
I think this 3rd assert can be removed given we already assert what the expected full string of the valid hashable_text is, so don't need to test if BAD is not in it